### DISCUSSION TITLE
"Following" tab is not limited to who I follow

### DISCUSSION THREAD

**Discussion by PeterNjeim** [community/community#31519] [Other Feature Feedback, Questions, & Ideas]
I wish I could follow people and see what they're up to, but unfortunately I have to scroll through endless feed items telling me someone has forked or starred my repo. This hinders my ability to see the things I follow. The "Following" tab should be for who I follow only. The stars and forks don't even need to be in any feed at all in reality, I can occasionally check the stargazers if I desire knowing the identities of people who star my repos.

**Comment by riversdiniz** [comment]
It's really complicated.
